lamptest.cpp

```diff
@@ -184,6 +195,19 @@ void LampTest::start()
     // Notify PHYP to start the lamp test
     doHostLampTest(true);
 
+    std::fstream lampTestFileDesc;
```

Comment:
Why close if not opened?

Also why not just std::ofstream (LAMP_TEST_IDICATOR_FILE); will create the file

Reply:
>Also why not just std::ofstream (LAMP_TEST_IDICATOR_FILE); will create the file
Even in this case the mode will be taken as "ios_base::out" by default? and that is why we will have to close it?